### PR TITLE
Fix/26

### LINKED ISSUE #26: Ruby warning
Hi,

When using `check-jenkins-job-status.rb`check, i received ruby warning message about `rbreadline`:

`/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/rb-readline-0.5.3/lib/rbreadline.rb:13: warning: constant ::Fixnum is deprecated `

About the setting:
- Debian 8
- Sensu 1.0.2

Could you fix it please ?

Thanks

### PR DESCRIPTION
**Pull request by majormoses**
## Pull Request Checklist

fixes #26 

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass 

#### Purpose
Remove deprecation warnings about `Fixnum` by updating depedencies
#### Known Compatibility Issues
none

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,15 +4,15 @@
 
 #### General
 
-- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)
+- [ ] Update Changelog following the conventions laid out on [Our CHANGELOG Guidelines ](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
```
Comment (eheydrick) [inline .github/PULL_REQUEST_TEMPLATE.md]: extra space.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,19 @@
 #Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
+This CHANGELOG follows the format listed at [Our CHANGELOG Guidelines ](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md).
```
Comment (eheydrick) [inline CHANGELOG.md]: extra space

**Review by eheydrick — COMMENTED** [review]
Fix looks good. Just formatting.

**Comment by majormoses** [comment]
@eheydrick fixed.

**Review by eheydrick — APPROVED** [review]

**Pull request merged**